### PR TITLE
bounty gaming changes

### PR DESCRIPTION
**Pull request by whateverusername0**
## About the PR

image

ok :trollface: 
also added ling action borders because they look cool
and some placeholders for regrow ability that will regrow ling limbs

now we wait for the great shitmed refactor

## Why / Balance

dynamic needs to die and i killed it.

:cl:
- fix: Fixed dynamic.
- add: Added borders to changeling's abilities.

### CODE REVIEW

File: Content.Server/Administration/PlayerLocator.cs [hunk 1]
```diff
@@ -106,6 +106,9 @@ public PlayerLocator()
 
         public async Task<LocatedPlayerData?> LookupIdAsync(NetUserId userId, CancellationToken cancel = default)
         {
+#if DEBUG
+            return null; // this is MF DOOM.
+#endif
```
[the remark this answers is not in the document]
Comment (whateverusername0) [reply Content.Server/Administration/PlayerLocator.cs:111]: if i don't do that it will guaranteedly shit out a SSL error so i had to pick the lesser evil

**Comment by JustSome1Nearby** [comment]
Can you merge the fixed dynamic?

**Comment by Piras314** [comment]
@whateverusername0 can you merge master into this and make sure it's still working after upstream merge?

**Comment by whateverusername0** [comment]
ok

**Review by Piras314 — COMMENTED** [review]
vro just straight up deleted a ton of logic...

**Comment by Piras314** [comment]
whatever. go my self assign.

**Review by Piras314 — APPROVED** [review]
the round starts on dynamic, and this doesn't add any checkfails as far as I can see. good enough for me.

**Comment by whateverusername0** [comment]
> vro just straight up deleted a ton of logic...

the distraction caused the bugger...

**Pull request merged**